dsb-gui/src/main/java/com/spectralogic/dsbrowser/gui/services/tasks/RecoverInterruptedJob.java

```diff
@@ -170,7 +167,7 @@ public void executeJob() throws Exception {
                 })
                 .doOnComplete(() -> {
                     loggingService.logMessage(buildFinalMessage, SUCCESS);
-                    removeJobIdAndUpdateJobsBtn(jobInterruptionStore, uuid);
+                    //removeJobIdAndUpdateJobsBtn(jobInterruptionStore, uuid);
```

Comment:
Is there a reason this was commented out?

Reply:
I figured out my weird half thought. removeJobIdAndUpdateJobsBtn had no effect (see changes to Ds3JobTask) It should be totally deleted not just commented out.